src/component/TodoItem.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styled from "styled-components";
+import { BsPatchCheck, BsPatchCheckFill, BsEraserFill } from "react-icons/bs";
+
+const Button = styled.div`
+  border: 0;
+  background: none;
+  cursor: pointer;
+`;
+
+const Item = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 5px;
+  height: 30vh;
```

Comment:
요 30vh부분이 너무 커서 아이템이 카드 전체 영역을 차지하도록 렌더링 되는 것 같아요!
<img width="342" alt="스크린샷 2023-03-25 오후 3 20 32" src="https://user-images.githubusercontent.com/77270547/227700556-fbea8f71-2084-4fca-a9ed-604eea8492a1.png">
<img width="342" alt="스크린샷 2023-03-25 오후 3 19 33" src="https://user-images.githubusercontent.com/77270547/227700567-01bd1104-0829-47f1-97cb-bf8fdeb1286a.png">
```suggestion
  height: 3vh;
```
정도로 해주는게 좋아보입니다!

Reply:
헛 그렇군요! 이 부분 수정하면 좋을 것 같아요 알려주셔서 감사합니다!😄